<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersEdiditing</class>
 <widget class="QWidget" name="UsersEdiditing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>203</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>419</width>
    <height>203</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>419</width>
    <height>203</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добваление пользователя</string>
  </property>
  <widget class="QLineEdit" name="loginLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="psswdLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addNewUserPushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>97</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="whichLine">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>ID записи, куда хотите добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="necessary">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="necessary_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
